jobboard/admin.py

```diff
@@ -1,3 +1,5 @@
-# from django.contrib import admin
+from django.contrib import admin
+from django.apps import apps
 
-# Register your models here.
+for model in apps.get_app_config("jobboard").get_models():
```

Comment:
this seems simpler then the code Sim gave you, but I still think that importing the individual models and registering each one would be simpler to understand in the long run, even if it does require slightly more maintainance

Reply:
We used this for loop technique for all models.py placement
In admin.py, also for the PR's that have already been approved. Is it bad practice? Or your personal preference?

Reply:
If I shown you this code without explaining what it does - how long would it take you to figure it out? How certain are you about the stability of the APIs you use there such as `get_app_config` and `get_models`?

I susually prefer a simpler solution until the need for a complex one shows up

Reply:
Ok , on it.